go.mod

```diff
@@ -18,3 +18,5 @@ require (
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+go 1.11
```

Comment:
We support go 1.9, as that's what client_golang supports.

Reply:
IMO it doesn't make sense to have a version number less than 1.11 since Go modules aren't available for previous versions.